src/pretalx/submission/models/review.py

```diff
@@ -40,11 +41,30 @@ def recalculate_scores(cls, event):
         for review in event.reviews.all():
             review.save(update_score=True)
 
+    @property
```

Comment:
What's the reason for this method to be a property? It's something that's called, and a verb, so semantically and logically it should be a method, imo.

Reply:
Oh! I get that the reason I added it as a property as 

> Added a def _validate_independence(self) method, marked it as property since if I used it without a property the function was returning a bound method error.

But will work on it how this can be made into a normal method. 😄